### PR TITLE
Atmos Manual, Digital Valves and Heat Exchange

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: [New]Adds manual and digital valves

CL: [New]Adds heat exchange

-Fix freezers, heaters, thermo machines not working

### CODE REVIEW

**Review by NoooneyDude — DISMISSED** [review]
Over in /tg/ we had to set up two exchanger machines, one for each pipenet. Apparently it was supposed to draw lines between them to represent fins or something too, but it's been a longstanding bug, it seems : P

File: UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs [hunk 1]
```diff
@@ -313,8 +323,9 @@ public enum PipeType
 	{
 		None = 0,
 		PipeRun = 1 << 0,
-		CoolingPipe = 1 << 1,
 		//Used for stopping cooling pipes to connect to usual pipes
+		CoolingPipe = 1 << 1,
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs]: I think this is what cooling pipe was meant to be

**Comment by corp-0** [comment]
solve the codacy issue and we are good to go

File: UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs (hunk 1 above)
Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs]: What do you mean by this?

**Review by NoooneyDude — DISMISSED** [review]
As Gilles refers to, an enum representing the possible sprites might be nice. Otherwise all good.

File: UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs]: That's what it originally meant to be. Heat Exchanger == CoolingPipe

**Comment by ThatDan123** [comment]
Sorry, just been a bit busy. Will do it tomorrow

File: UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Systems/Fluids/PipeFunctions.cs]: Fixed

**Review by Bod9001 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**